lldb/source/Commands/CommandObjectTarget.cpp

```diff
@@ -3494,6 +3506,7 @@ class CommandObjectTargetModulesShowUnwind : public CommandObjectParsed {
                                      // parsing options
     std::string m_str; // Holds name lookup
     lldb::addr_t m_addr = LLDB_INVALID_ADDRESS; // Holds the address to lookup
+    bool m_cached = false;
```

Comment:
I'm fine with changing the behavior of the existing command if you'd like to.  This command is very much only used by lldb maintainers and tests.